src/headers/ctrl.h

```diff
@@ -100,11 +100,12 @@ typedef struct Ctrl_struct {
     uint8_t payload[CTRL_MAX_PAYLOAD_SIZE];
 } Ctrl;
 
-typedef struct CtrlProfileName_struct {
+typedef struct CtrlProfileMeta_struct {
     // Must be packed (58 bytes).
     char name[24];
-    char padding[34];
-} CtrlProfileName;
+    uint32_t version;
+    uint8_t padding[33];
```

Comment:
Should probably be a padding[30], since you are going from a uint8 to a uint32.
How do you validate this?